dbt/adapters/__about__.py

```diff
@@ -1 +1 @@
-version = "1.8.0a2"
+version = "1.0.0a2"
```

Comment:
Revert version back to the correct one. This must have been inadvertently bumped to the `dbt-tests-adapter` version.